src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,89 @@
-export const Pagination = () => {};
+import classNames from 'classnames';
+
+interface Props {
+  items: string[],
+  pageNumber: number[],
+  paginate: (page: number) => void,
+  makeStep: (direction: string) => void,
+  pageNumberBack: number,
+}
+
+export const Pagination: React.FC<Props> = ({
+  items,
+  pageNumber,
+  paginate,
+  pageNumberBack,
+  makeStep,
+}) => {
+  function getMaxOfArray(numArray: number[]) {
```

Comment:
move it from the component

Reply:
I have moved getMaxOfArray to utils) it seemed logical